src/target/riscv/riscv-013.c

```diff
@@ -4851,6 +4851,19 @@ struct target_type riscv013_target = {
 static int riscv013_get_register(struct target *target,
 		riscv_reg_t *value, enum gdb_regno rid)
 {
+	/* This would be nice to move this redirection, to version-independent part,
```

Comment:
I would reword this:
```suggestion
	/* It would be beneficial to move this redirection to the version-independent section,
```

Reply:
Addressed.

Reply:
Thanks.